src/parser/pegen.ts

```diff
@@ -1528,6 +1539,20 @@ export function seq_flatten(p: Parser, seqs: AST[][]): AST[] {
 //     return new_seq;
 // }
 
+class CmpopExprPair {
+    cmpop: cmpop;
+    expr: expr;
+
+    constructor(cmpop: cmpop, expr: expr) {
+        this.cmpop = cmpop;
+        this.expr = expr;
+    }
+}
```

Comment:
We should export these and add the imports to the generated parser since they appear in the return types for some of the methods. 
Maybe move to another file - `pegen_types.ts`?